src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -2267,6 +2267,72 @@ paths:
 ##############################################################################
 ## SEARCH API STANDARD ENDPOINTS
 ##############################################################################
+  /api/repository/v1/search:
```

Comment:
I think this would be an API under `search` not at the root level. Maybe `/search/metadata` (to match the PUT and DELETE endpoints)?